build_tools/github_actions/configure_ci.py

```diff
@@ -360,15 +500,24 @@ def main():
         or LLVM_INTEGRATE_BRANCH_PATTERN.search(os.environ.get("PR_BRANCH", ""))
         or LLVM_INTEGRATE_LABEL in labels
     )
+
+    modifies = modifies_included_path()
+    try:
+        benchmark_presets = get_benchmark_presets(
+            trailers, labels, is_pr, is_llvm_integrate_pr
+        )
+        all_jobs = parse_jobs_from_workflow_file()
+        enabled_jobs = get_enabled_jobs(modifies, is_pr, trailers, all_jobs)
```

Comment:
Ah... the order of arguments in this call is wrong. Now CI is not running in postsubmit because `modifies` is wrongly passed to the parameter `is_pr`, which is always `True` in postsubmit.

Reply:
:cry: this is why boolean args are bad